docs/products/postgresql/concepts/upgrade-failover.rst

```diff
@@ -57,3 +57,29 @@ During maintenance updates, cloud migrations, or plan changes, the below procedu
 
 .. Note::
     The old primary server is kept alive for a short period of time (minimum 60 seconds) with a TCP forwarding setup pointing to the new primary server allowing clients to connect before learning the new IP address.
+
+Recreation of replication slots
```

Comment:
Should we have a dedicated article for this? It's a pretty important (and not usual) setup on our side

Reply:
@divdby0 @alexole do you think we need a separate article for Recreation of replication slots? Or is the section in this article ok?

Reply:
I understand @ftisiot 's point that this is a valuable feature and requires a bit more highlighting. However, I also believe that this is categorically the correct place for this feature. Because most of the importance of this feature comes from its recent built-up customer expectations. I believe, over time it will be normalized, and this feature will be a standart, expected, intuitive behavior, similarly of our existing handling of the physical replication slots.

Note that we have already been in touch lately with some of the customers via the CSMs, and we'll continue to use the CSM channel to promote this feature to the customers that demanded have this feature in the past.

Reply:
Even though the complexity of the feature might deserve a separate article, from customer POV it should have worked already :) So I would just leave it here.